test/status_test.go

```diff
@@ -45,6 +47,8 @@ var (
 	embeddedStatusFlag = requireAllGates(map[string]string{
 		"embedded-status": "full",
 	})
+
+	ignoreFeatureFlags = cmpopts.IgnoreFields(v1beta1.Provenance{}, "FeatureFlags")
```

Comment:
Why are we ignoring feature flags in the tests?

Reply:
Because the exact values of the feature flags can change depending on what is invoked on the cluster. Therefore, I'm testing that they are not `nil` instead of checking individual feature flags which could change depending on whether the CI is running `alpha`, `beta` or `stable` feature flags.